src/source/any_source_component.ts

```diff
@@ -21,6 +22,7 @@ export const any_source_component = z.union([
   source_simple_power_source,
   source_simple_battery,
   source_simple_inductor,
+  source_simple_pushbutton,
```

Comment:
Please change to push_button

Reply:
Ok

Reply:
Clarification: the full name should be simple_source_push_button